### PR TITLE
Changed all 'NED board' references to 'FRD body'. Also cleaned up mix…

### PR DESCRIPTION
**Pull request by rblilja**
…ing of m/s/s and m/s^2 to use the latter. There are still some messages that needs to be looked at for correct reference frame: 'vehicle_rates_setpoint.msg' and 'vehicle_attitude_setpoint.msg

Solves issue: #16328

### CODE REVIEW

**Comment by dagar** [comment]
There are conflicts in quite a few files, can you rebase this on current master?

File: msg/sensor_correction.msg [hunk 1]
```diff
@@ -21,29 +21,29 @@ float32[3] gyro_scale_2		# gyro XYZ scale factors in the sensor frame
...
 # corrections for uORB instance 2
-float32[3] accel_offset_2	# accelerometer XYZ offsets in the sensor frame in m/s/s
+float32[3] accel_offset_2	# accelerometer XYZ offsets in the sensor frame in m/s^2
 float32[3] accel_scale_2	# accelerometer XYZ scale factors in the sensor frame
 
 # Corrections for barometric pressure outputs where corrected_pressure = raw_pressure * pressure_scale + pressure_offset
 # Note the corrections are in the sensor frame and must be applied before the sensor data is rotated into body frame
 # corrections for uORB instance 0
-float32 baro_offset_0		# barometric pressure offsets in the sensor frame in m/s/s
+float32 baro_offset_0		# barometric pressure offsets in the sensor frame in m/s^2
```
Comment (dagar) [inline msg/sensor_correction.msg:38]:
```suggestion
float32 baro_offset_0		# barometric pressure offsets in the sensor frame in Pascals.
```

**Comment by rblilja** [comment]
> There are conflicts in quite a few files, can you rebase this on current master?

@dagar I added current master as upstream and rebased my fork. Then pushed my origin to my fork master. Does that suffice? Sorry - I'm new to rebasing.

**Comment by dagar** [comment]
> > There are conflicts in quite a few files, can you rebase this on current master?
> 
> @dagar I added current master as upstream and rebased my fork. Then pushed my origin to my fork master. Does that suffice? Sorry - I'm new to rebasing.

Yes that would be fine, but it doesn't look like that branch made it to github as it still reports the conflicts.

Screen Shot 2020-12-06 at 1 32 13 PM

Merging with latest PX4 master is also fine.

**Comment by rblilja** [comment]
@dagar Got messy. Closing this one and making a fresh.

**Pull request closed without merging**